pdc_client/__init__.py

```diff
@@ -120,16 +120,18 @@ class to perform requests.
             except KeyError:
                 print("'%s' must be specified in configuration file." % CONFIG_URL_KEY_NAME)
                 sys.exit(1)
-            if not isinstance(ssl_verify, str):
-                ssl_verify = config.get(CONFIG_SSL_VERIFY_KEY_NAME, ssl_verify)
-            insecure = config.get(CONFIG_INSECURE_KEY_NAME)
-            if insecure is not None:
-                sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
-                    CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
-                if insecure:
-                    ssl_verify = False
-                elif not insecure and not ssl_verify:
-                    ssl_verify = True
+            if ssl_verify is None:
+                ssl_verify = config.get(CONFIG_SSL_VERIFY_KEY_NAME)
+                insecure = config.get(CONFIG_INSECURE_KEY_NAME)
+                if insecure is not None:
+                    sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
+                        CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
+                    if insecure:
+                        ssl_verify = False
+                    elif not insecure and not ssl_verify:
+                        ssl_verify = True
+            elif ssl_verify is True:
+                ssl_verify = False
```

Comment:
So `insecure` is false mean `ssl_verify` is false here. Why ssl_verify should be the same as insecure value here?